src/main/java/org/openjdk/jextract/impl/Constants.java

```diff
@@ -432,47 +427,30 @@ private Constant emitFunctionDescField(FunctionDescriptor desc) {
         }
 
         private Constant emitConstantString(Object value) {
-            incrAlign();
-            indent();
-            append(MEMBER_MODS);
-            append(" MemorySegment ");
             NamedConstant segConstant = new NamedConstant(MemorySegment.class);
-            append(segConstant.constantName);
-            append(" = RuntimeHelper.CONSTANT_ALLOCATOR.allocateFrom(\"");
-            append(Utils.quote(Objects.toString(value)));
-            append("\");\n");
-            decrAlign();
+            appendIndentedLines(STR."""
```

Comment:
the `emitLayoutString` method could use a similar treatment - at least for some aspects?

Reply:
I've sprinkled some more use of interpolation.